.github/workflows/user_acceptance_tests.yml

```diff
@@ -9,11 +9,11 @@ jobs:
   user-acceptance-tests:
     runs-on: ubuntu-latest
     
+    
     strategy:
       fail-fast: false
       matrix:
-        cloud_provider: [aws, azure, gcp] 
-        scenario: [happy, mysql]
+        uat_file_name: ["happy-aws-debian.deploy.config.json","happy-aws-linux2.deploy.config.json","happy-azure.deploy.config.json","happy-gcp.deploy.config.json","mysql-aws.deploy.config.json","mysql-azure.deploy.config.json","mysql-gcp.deploy.config.json"]
```

Comment:
Is it possible to multiline this? I can't remember if YAML lets you do that

Reply:
yea yaml has multiline array syntax. i believe its indent w/ dashes under the field.

Reply:
based on the most recent run, what i said appears to be incorrect 🙈

Reply:
@nr-kkenney I got it working!  I think there was a change on the GitHub side of life.  Take a look at the most recent commit to this PR.

Reply:
https://github.com/newrelic/demo-javatron/pull/14/commits/4fa8046d7e686682bd3d1f83e85b7d09928d68d2